pkg/kv/kvserver/raftlog/entry.go

```diff
@@ -33,33 +33,33 @@ import (
 func EncodingVersion(ent raftpb.Entry) (EntryEncoding, error) {
 	if len(ent.Data) == 0 {
 		// An empty command.
-		return RaftVersionEmptyEntry, nil
+		return EntryEncodingEmpty, nil
 	}
 
 	switch ent.Type {
 	case raftpb.EntryNormal:
 	case raftpb.EntryConfChange:
-		return RaftVersionConfChange, nil
+		return EntryEncodingRaftConfChange, nil
 	case raftpb.EntryConfChangeV2:
-		return RaftVersionConfChangeV2, nil
+		return EntryEncodingRaftConfChangeV2, nil
 	default:
 		return 0, errors.AssertionFailedf("unknown EntryType %d", ent.Type)
 	}
 
 	switch ent.Data[0] {
-	case RaftVersionStandardPrefixByte:
-		return RaftVersionStandard, nil
-	case RaftVersionSideloadedPrefixByte:
-		return RaftVersionSideloaded, nil
+	case EntryEncodingStandardPrefixByte:
+		return EntryEncodingStandard, nil
+	case EntryEncodingSideloadedPrefixByte:
+		return EntryEncodingSideloaded, nil
 	default:
 		return 0, errors.AssertionFailedf("unknown command encoding version %d", ent.Data[0])
 	}
 }
 
 // DecomposeRaftVersionStandardOrSideloaded extracts the CmdIDKey and the
 // marshaled kvserverpb.RaftCommand from a slice which is known to have come
-// from a raftpb.Entry of type kvserverbase.RaftVersionStandard or
-// kvserverbase.RaftVersionSideloaded (which, mod the prefix byte, share an
+// from a raftpb.Entry of type kvserverbase.EntryEncodingStandard or
+// kvserverbase.EntryEncodingSideloaded (which, mod the prefix byte, share an
```

Comment:
The comment needs update, the values moved to a different package. Consider not spelling out the package, it should be clear from the context.

Reply:
If this were just text I wouldn't spell them out, but if you put the package with the type then Goland will allow you to navigate to it right from the comment, which is useful.

Reply:
It's the same package, so it should still navigate without a package prefix (I think).

Reply:
Oh, got lost in all the renames. Yes, you're right. Didn't realize this was in the same package.